index.html

```diff
@@ -14,256 +14,258 @@
-                        <li><a href="#"><img src="/photos/mail.jpg" alt=""></a></li>
-                    </ul>
-                </div>
-            </div>
-            <footer class="submit">
-                <button type="submit">Get my resume</button>
-            </footer>
-        </section>
-        <section class="Skills">
-            <div class="Languages">
-                <div class="lang">
-                    <h3>Languages</h3>
-                </div>
-                <div class="script">
-                    <img src="/photos/javasvript.png" alt="">
-                    <p>JavaSript</p>
+            <section class="Skills">
+                <div class="Languages">
+                    <div class="lang">
+                        <h3>Languages</h3>
+                    </div>
+                    <div class="script">
+                        <img src="/photos/javasvript.png" alt="">
+                        <p>JavaSript</p>
+                    </div>
+                    <div class="script">
+                        <img src="/photos/html5.png" alt="">
+                        <p>HTML</p>
+                    </div>
+                    <div class="script">
+                        <img src="/photos/css3.png" alt="">
+                        <p>CSS</p>
+                    </div>
                 </div>
-                <div class="script">
-                    <img src="/photos/html5.png" alt="">
-                    <p>HTML</p>
+                <div class="frame">
+                    <p>Frameworks</p>
+                    <img src="/photos/separator.png" alt="">
                 </div>
-                <div class="script">
-                    <img src="/photos/css3.png" alt="">
-                    <p>CSS</p>
+                <div class="ski">
+                    <p>Skills</p>
+                    <img src="/photos/separator.png" alt="">
                 </div>
+            </section>
+        </div>
+    
+        <div class="l-container">
+            <div class="contact">
+                <h1>Contact me</h1>
+                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs
+                coding. I’d love to help with it</p>
             </div>
-            <div class="frame">
-                <p>Frameworks</p>
-                <img src="/photos/separator.png" alt="">
-            </div>
-            <div class="ski">
-                <p>Skills</p>
-                <img src="/photos/separator.png" alt="">
+            <div class="form">
+                <form action="GET">
+                    <input type="text" name="name" placeholder="Merianne">
+                    <br>
+                    <input type="email" name="email" placeholder="natajayanj14@gmail.com">
+                    <br>
+                    <textarea name="name"  cols="38" rows="10">Write your message here</textarea>
```

Comment:
- [ ] KIndly ensure that your textarea input has a required attribute. This will add form validation to ensure the user fills information  into the text area
- [ ] KIndly ensure that your textarea has a `maxlength="500"` attribute. This will stop users from filling in too much info here